cmd/podman/kube/generate.go

```diff
@@ -81,9 +81,6 @@ func generateFlags(cmd *cobra.Command, podmanConfig *entities.PodmanConfig) {
 	flags.Int32VarP(&generateOptions.Replicas, replicasFlagName, "r", 1, "Set the replicas number for Deployment kind")
 	_ = cmd.RegisterFlagCompletionFunc(replicasFlagName, completion.AutocompleteNone)
 
-	noTruncAnnotationsFlagName := "no-trunc"
-	flags.BoolVar(&generateOptions.UseLongAnnotations, noTruncAnnotationsFlagName, false, "Don't truncate annotations to Kubernetes length (63 chars)")
-
```

Comment:
Deprecating before obsoleting makes sense. There were two thoughts for I removed it completely.
1. This is a “human” flag (as mentioned in the source code), not meant for automation.
2. Version 5.0 is introducing several other breaking changes.

So, this seemed the right time. Let me know if you still insist.

Reply:
Regarding (1), not sure what it means. I have a bunch of Quadlet files ready for using and I guess others may have Quadlets or shell scripts that will now break. Having said that, (2) is a good point.
@vrothberg is usually the person to ask this. WDYT?

Reply:
Regarding 1): it's shipped, so removing it will remain a breaking change.

Regarding 2): I let @mheon and @baude chime in whether the time window is still open for breaking changes.

Reply:
Marked hidden and deprecated